<launch>
  <arg name="station_id" default="0"/>
  <!-- <group ns="swarm_bridge"> -->
    <node pkg="swarm_bridge" name="station_bridge_node_tcp" type="test_tcp_station" output="screen">
      <!-- <remap from="/occupancy_inflate_from_bridge" to="occupancy_inflate"/>
      <remap from="/others_odom1" to="/drone_1/drone_odom_raw"/>
      <remap from="/others_odom2" to="/drone_2/drone_odom_raw"/>
      <remap from="/others_odom3" to="/drone_3/drone_odom_raw"/>
      <remap from="/others_odom4" to="/drone_4/drone_odom_raw"/>
      <remap from="/others_odom5" to="/drone_5/drone_odom_raw"/>


      <remap from="/others_odom_raw1" to="/drone_1/vel_odom"/>
      <remap from="/others_odom_raw2" to="/drone_2/vel_odom"/>
      <remap from="/others_odom_raw3" to="/drone_3/vel_odom"/>
      <remap from="/others_odom_raw4" to="/drone_4/vel_odom"/>
      <remap from="/others_odom_raw5" to="/drone_5/vel_odom"/> -->

      <!-- <remap from="~my_so2_cmd1" to="/drone_1/so2_cmd"/>
      <remap from="~my_so2_cmd2" to="/drone_2/so2_cmd"/>
      <remap from="~my_so2_cmd3" to="/drone_3/so2_cmd"/>
      <remap from="~my_so2_cmd4" to="/drone_4/so2_cmd"/>
      <remap from="~my_so2_cmd5" to="/drone_5/so2_cmd"/> -->
      <remap from="~my_cmd_vel1" to="/drone_1/cmd_vel"/>
      <remap from="~my_cmd_vel2" to="/drone_2/cmd_vel"/>
      <remap from="~my_cmd_vel3" to="/drone_3/cmd_vel"/>
      <remap from="~my_cmd_vel4" to="/drone_4/cmd_vel"/>
      <remap from="~my_cmd_vel5" to="/drone_5/cmd_vel"/>
      <!-- <remap from="/share_odom1" to="/share_odom"/>
      <remap from="/share_odom2" to="/share_odom"/>
      <remap from="/share_odom3" to="/share_odom"/>
      <remap from="/share_odom4" to="/share_odom"/>
      <remap from="/share_odom5" to="/share_odom"/> -->

      <!-- <param name="so2_cmd_broadcast_freq_" value="100"/> -->
      <param name="cmd_vel_broadcast_freq_" value="50"/>

      <param name="self_id" value="$(arg station_id)"/>
      <param name="is_ground_station" value="true"/>
      <param name="drone_num" value="8"/>
      <param name="drone_ip_0" value="192.168.1.21" type="string"/>
      <param name="drone_ip_1" value="192.168.50.21" type="string"/>
      <param name="drone_ip_2" value="192.168.50.22" type="string"/>
      <param name="drone_ip_3" value="192.168.50.23" type="string"/>
      <param name="drone_ip_4" value="192.168.50.24" type="string"/>
      <param name="drone_ip_5" value="192.168.50.25" type="string"/>
      <param name="drone_ip_6" value="192.168.50.26" type="string"/>
      <param name="drone_ip_7" value="192.168.50.27" type="string"/>


      <param name="ground_station_num" value="1"/>
      <param name="ground_station_ip_0" value="192.168.1.100" type="string"/>

      <param name="pub_occupancy" value="true" type="bool"/>
      <param name="pub_so2_cmd" value="true" type="bool"/>
      <param name="pub_cmd_vel" value="true" type="bool"/>
    </node>

</launch>
